api/nodes/views.py

```diff
@@ -152,6 +159,48 @@ def get_queryset(self):
         return self.get_queryset_from_request()
 
 
+class NodeContributorDetail(generics.RetrieveUpdateDestroyAPIView, NodeMixin):
+    """Detail of a contributor for a node.
+
+    Contributors are users who can make changes to the node or, in the case of private nodes,
+    have read access to the node. Contributors are divided between 'bibliographic' and 'non-bibliographic'
+    contributors. From a permissions standpoint, both are the same, but bibliographic contributors
+    are included in citations, while non-bibliographic contributors are not included in citations.
+
+    Allows for a user to view, remove a contributor from, and change bibliographic and admin status for a node.
```

Comment:
This sentence could use a revision to make it easier to parse, but it's fine for now.